src/main/java/com/hrznstudio/galacticraft/server/command/GalacticraftCommands.java

```diff
@@ -23,119 +23,251 @@
+             * teleporting entities OR setting a custom position to go to, but not both :P
+             */
+            LiteralCommandNode<ServerCommandSource> dimensiontp_root = commandDispatcher.register(
+                    LiteralArgumentBuilder.<ServerCommandSource>literal("dimensiontp")
+                    .requires(serverCommandSource -> serverCommandSource.hasPermissionLevel(2))
+                    .then(CommandManager.argument("dimension", DimensionArgumentType.dimension())
+                    .executes(GalacticraftCommands::teleport)));
+            // TODO: either fix this or remove it
+            /* LiteralCommandNode<ServerCommandSource> dimensiontp_entities = commandDispatcher.register(
+                    LiteralArgumentBuilder.<ServerCommandSource>literal("dimensiontp")
+                    .requires(serverCommandSource -> serverCommandSource.hasPermissionLevel(2))
+                    .then(CommandManager.argument("dimension", DimensionArgumentType.dimension())
+                    .then(CommandManager.argument("entities", EntityArgumentType.entities())
+                    .executes(((GalacticraftCommands::teleportMultiple)))))); */
+            LiteralCommandNode<ServerCommandSource> dimensiontp_pos = commandDispatcher.register(
+                    LiteralArgumentBuilder.<ServerCommandSource>literal("dimensiontp")
                     .requires(serverCommandSource -> serverCommandSource.hasPermissionLevel(2))
                     .then(CommandManager.argument("dimension", DimensionArgumentType.dimension())
-                            .executes(GalacticraftCommands::teleport)
-                            .then(CommandManager.argument("entities", EntityArgumentType.entities())
-                                    .executes(((GalacticraftCommands::teleportMultiple))))));
-            commandDispatcher.register(LiteralArgumentBuilder.<ServerCommandSource>literal("gcr_listbodies")
-                    .executes(context -> {
-                        StringBuilder builder = new StringBuilder();
-                        CelestialBodyType.getAll().forEach(celestialBodyType -> builder.append(celestialBodyType.getTranslationKey()).append("\n"));
-                        context.getSource().sendFeedback(new LiteralText(builder.toString()), true);
-                        return 1;
-                    }));
+                    .then(CommandManager.argument("pos", BlockPosArgumentType.blockPos())
+                    .executes(GalacticraftCommands::teleportToCoords))));
+
+            // Because I don't like to type
+            commandDispatcher.register(LiteralArgumentBuilder.<ServerCommandSource>literal("dimtp").redirect(dimensiontp_root));
+            //commandDispatcher.register(LiteralArgumentBuilder.<ServerCommandSource>literal("dimtp").redirect(dimensiontp_entities));
+            commandDispatcher.register(LiteralArgumentBuilder.<ServerCommandSource>literal("dimtp").redirect(dimensiontp_pos));
```

Comment:
Use `CommandManager#literal` instead of `LiteralArgumentBuilder#literal` to avoid explicitly specifying generics